pkg/controller/v1beta1/inferenceservice/reconcilers/ingress/kube_ingress_reconciler.go

```diff
@@ -100,11 +100,11 @@ func generateMetadata(isvc *v1beta1api.InferenceService,
 	var name string
 	switch componentType {
 	case constants.Transformer:
-		name = constants.DefaultTransformerServiceName(isvc.Name)
+		name = constants.TransformerServiceName(isvc.Name)
```

Comment:
@Suresh-Nakkeran will this change break the existing inference service deployed in raw kubernetes mode?

Reply:
Yes @yuzisun .. I tested. It will break the existing inference service if it is deployed in raw kubernetes mode.

Reply:
I am working on the fixes, will update the PR